Python_prog/addition.py

```diff
@@ -0,0 +1,5 @@
+# program to take two number as input and print sum of it
+num1=int(input("Enter a number1: "))
+num2=(int(input("Enter number2: ")))
```

Comment:
Why extra braces?

Reply:
sorry sir,might i have added it by mistake after running